### ISSUE TITLE
2024-11 Public Service Announcement (PSA) - todo.txt project effectively unmaintained

### ISSUE THREAD

**Issue by clach04** [todotxt/todo.txt#100]
I'm super grateful for this specification but for anyone who spots problems or has ideas for enhancements, be aware that based on the past few years I believe it unlikely any changes will be merged into the official standard. A number of minor (clarifying) changes that don't fundamentally change the spec have not been merged, hence why I think it unlikely. The original maintainers likely don't have the time/energy to work on this any more.

If you have something cool to share, definitely post here, there are a number of people who are still interested, monitoring, and participating who'll likely be interested in what you have to share.

If you really think you have something new on the standards front, I recommend you fork/make a new standard with a **different** name to avoid name clashes (you can still retain todo.text compatibility). It's not something to take on likely, you'll likely have a lot of interest 😀. I personally don't have the time/energy to do that.

Definitely read through the existing PRs and issues 😉 

Again, thank you to everyone who participates! ❤️  The discussion in issues and PRs has helped me understand a few nuances of the spec that could not be gleaned from the spec as it was initially written.

**Comment by hyperupcall** [comment]
It definitely possible for this project to get back up and running, but it's not quite there. A month or so there was a bit of activity on the code repository (I have had a few open PRs for at least a year or two now), but not much since. Related issue: https://github.com/todotxt/todo.txt/issues/80

**Comment by clach04** [comment]
> It definitely possible for this project to get back up and running, but it's not quite there. A month or so there was a bit of activity on the code repository (I have had a few open PRs for at least a year or two now), but not much since. Related issue: #80

I'm assuming you mean the https://github.com/todotxt/todo.txt-cli repo? As of the time I'm replying, the last change to this repo was 3 years ago https://github.com/todotxt/todo.txt/commits/master/ in 2021 and no other active branches https://github.com/todotxt/todo.txt/branches/all

I can't see any PRs with your user name on this repo https://github.com/todotxt/todo.txt/issues?q=+mentions%3Ahyperupcall+ there are open PRs from a few years ago so you are not alone in waiting for responses.

The linked issue you referred to is one of the reasons I posted this, I was involved in that discussion and I'm less optimistic now.

Let's keep our fingers crossed 🤞 to be clear the maintainers don't owe us anything kand I'm grateful for this spec and the work already done), I want to make sure people are clear on status without having to guess or spend a lot of time researching old issues/PRs

**Comment by clach04** [comment]
My attempt to handle my own OCD ;-) consolidated changes into https://github.com/too-much-todotxt/spec/blob/main/The-Todo.txt-Format.md

I had time this weekend to do this. I'm not sure I'll have time to continue, so this was mostly for myself.

**Comment by Zevan770** [comment]
yes, @karbassi is working on https://github.com/karbassi/ticktick-cli.
